beets/util/artresizer.py

```diff
@@ -219,9 +231,12 @@ def _can_compare(self):
     @staticmethod
     def _check_method():
         """Return a tuple indicating an available method and its version."""
-        version = get_im_version()
-        if version:
-            return IMAGEMAGICK, version
+        try:
+            version, im_legacy = get_im_version()
+            if version:
+                return IMAGEMAGICK, version, im_legacy
```

Comment:
Following up on a comment below, the return value of `get_im_version` should probably be `None` (i.e., not a pair) when the command can't be found at all. This is a little annoying, but it means this code would look like:

```py
v = get_im_version()
if v:
    version, im_legacy = v
    ...
```

Reply:
The current try/except block already handles a None return, since trying to unpack None to two variables results in a TypeError, which we can `except` and `pass`. However, your approach is once again slightly shorter and more readable, so we'll go with that!